### PR TITLE
Add footer at page.blade.php

### PR DESCRIPTION
**Pull request by gabrielalinck**
I did noticed that the admin-lte page doesn't have a footer, so i put a @yield('footer') on page.blade.php. This way i can create my on footer on my admin-lte page.
I hope this helps.

### CODE REVIEW

**Comment by fsavina** [comment]
you should probably wrap the footer in a `@hasSection('footer')` block

File: resources/views/page.blade.php [hunk 1]
```diff
@@ -125,6 +125,10 @@
             @endif
         </div>
         <!-- /.content-wrapper -->
+        
+        <footer class="main-footer">
```
Comment (fredochieng) [inline resources/views/page.blade.php]: After adding this, I get a footer but it is blank and I dont know where to edit my footer

**Comment by lacodimizer** [comment]
@gabrielalinck Can you add the footer to a section please?

File: resources/views/page.blade.php (hunk 1 above)
Comment (lacodimizer) [reply resources/views/page.blade.php]: You can edit the footer global in the page.blade.php or you can overwrite the footer on every page.

**Pull request merged**